fenix/app/src/main/res/values/dimens.xml

```diff
@@ -11,6 +11,10 @@
     <dimen name="mozac_browser_menu_width_max" tools:ignore="UnusedResources">314dp</dimen>
     <dimen name="mozac_browser_menu_corner_radius">8dp</dimen>
     <dimen name="mozac_browser_menu2_corner_radius">8dp</dimen>
+    <dimen name="mozac_browser_menu_item_checkbox_container_padding_end" tools:ignore="UnusedResources">0dp</dimen>
+    <dimen name="mozac_browser_menu_item_image_checkbox_padding_end" tools:ignore="UnusedResources">16dp</dimen>
+    <dimen name="mozac_browser_menu_item_checkbox_text_label_padding_end" tools:ignore="UnusedResources">8dp</dimen>
+    <dimen name="mozac_browser_menu_item_image_checkbox_padding_start" tools:ignore="UnusedResources">8dp</dimen>
```

Comment:
```suggestion
    <dimen name="mozac_browser_menu_item_image_checkbox_padding_start" tools:ignore="UnusedResources">8dp</dimen>
    <dimen name="mozac_browser_menu_item_image_checkbox_padding_end" tools:ignore="UnusedResources">16dp</dimen>
```

Could we reorder these so that the image checkbox paddings are next to eachother and the same for the checkbox text label paddings?